samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/CallingBot.cs

```diff
@@ -142,6 +151,25 @@ private async Task NotificationProcessor_OnNotificationReceivedAsync(Notificatio
 
                 var recordingLocation = await teamsRecordingService.DownloadRecording(recording.RecordingLocation, recording.RecordingAccessToken);
 
+                try
```

Comment:
Update the comment above (line 144)

Reply:
Actually, I suppose we should just add a comment here, since the other one is still correct (we echo it back to the user)